unstructured/documents/html.py

```diff
@@ -407,9 +416,12 @@ def _find_main(root: etree.Element) -> etree.Element:
     return main_tag_elem if main_tag_elem is not None else root
 
 
-def _find_articles(root: etree.Element) -> List[etree.Element]:
+def _find_articles(root: etree.Element, assemble_articles: bool = True) -> List[etree.Element]:
```

Comment:
could there be an auto mode which does 
`root.findall(".//article")` if `article`s exist, otherwise use the root?

Reply:
i left this comment on the wrong function, but idea generally holds. Ideally, `partition_html` does the opinionated right thing by default for news-page html pages vs. plain old html pages.

Reply:
That's what it does now actually, it first looks for `.//article`, then `.//div[@itemprop='articleBody']` and if it doesn't find either then the returns `root`.